haxe.YAML-tmLanguage

```diff
@@ -637,30 +637,27 @@ repository:
     name: meta.parameters.hx
     patterns:
     - include: '#parameter'
-    - include: '#punctuation-comma'
```

Comment:
we should keep this line, just the changes to `parameter:` were not doing anything in my tests.